CHANGELOG.md

```diff
@@ -4,6 +4,84 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.10](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.10) - 2022-05-19
+
+Zebra's latest beta finalizes the support for `lightwalletd` RPC methods and continue with the testing of each of this features. Also, this beta set the NU5 mainnet activation height.
+
+### Added
+
+#### RPC
+
+- `z_gettreestate` RPC (#3990)
+
+##### Tests
+
+- grpc test for `GetTaddressBalanceStream` and `GetAddressUtxosStream` (#4407)
+- snapshot tests for RPC methods responses (#4352 #4401)
+
+#### CI
+
+- `lightwalletd_update_sync` test to CI (#4269)
+- `lightwalletd_full_sync` test to CI (#4268)
+
+### Changed
+
+- Set NU5 mainnet activation height and current network protocol version (#4390)
+- NU5 mainnet dependency upgrades (#4405)
+- Use the latest lightwalletd version (#4398)
+
+#### Dependency upgrades
+
+- google-github-actions/auth from 0.7.2 to 0.7.3 (#4419)
+- vergen from 7.0.0 to 7.1.0 (#4420)
+- tokio-util from 0.7.1 to 0.7.2 (#4406)
+- google-github-actions/auth from 0.7.1 to 0.7.2 (#4404)
+- tj-actions/changed-files from 19 to 20 (#4403)
+- w9jds/firebase-action from 2.0.0 to 2.1.0 (#4402)
+- inferno from 0.11.2 to 0.11.3 (#4357)
+- tokio from 1.18.1 to 1.18.2 (#4358)
+- prost from 0.10.2 to 0.10.3 (#4348)
+- from 0.8.1 to 0.9.0 (#4394)
```

Comment:
```suggestion
- bech32 0.8.1 to 0.9.0 (#4394)
```

Reply:
done as part of https://github.com/ZcashFoundation/zebra/pull/4430/commits/7571d7b9c80c75a2ff61831b70c87a3431cc6c39